utils/doclint/generateDotnetApi.js

```diff
@@ -521,7 +521,10 @@ function renderMethod(member, parent, output, name) {
     pushArg(argType, argName, arg);
   };
 
-  member.args.forEach(parseArg);
+  Array.from(member.args)
+    .map(i => i[1])
+    .sort((a, b) =>  b.alias ==='options' ? -1 : 0) //move options to the back to the arguments list
+    .forEach(parseArg);
```

Comment:
```suggestion
  member.argsArray
    .sort((a, b) =>  b.alias === 'options' ? -1 : 0) //move options to the back to the arguments list
    .forEach(parseArg);
```
argsArray contains the members as an array already. Should also work.